mephisto/abstractions/providers/mturk/mturk_unit.py

```diff
@@ -162,6 +162,8 @@ def get_status(self) -> str:
         requester = self.get_requester()
         client = self._get_client(requester._requester_name)
         hit = get_hit(client, mturk_hit_id)
+        if not hit:
+            return AssignmentState.SOFT_REJECTED
```

Comment:
Nit: The more I think about it, the more `AssignmentState.EXPIRED` is more appropriate for this.